tests/dags/common/loader/test_reporting.py

```diff
@@ -0,0 +1,26 @@
+from unittest import mock
+
+import pytest
+from common.loader.reporting import report_completion
+
+
+@pytest.fixture(autouse=True)
+def send_message_mock() -> mock.MagicMock:
+    with mock.patch("common.slack.send_message") as SendMessageMock:
+        yield SendMessageMock.return_value
+
+
+@pytest.mark.parametrize(
+    "should_send_message",
+    [
+        (True,),
+        (False,),
+    ],
```

Comment:
For multiple parameters in a single `parametrize`, tuples are necessary. If there's only one parameter though, this can be simplified to: 
```suggestion
    [True, False],
```

Reply:
Oh yikes, left if like this after removing another parameter -- that doesn't look great 😆 Updated!